tools/topology/sof/pipe-volume-demux-playback.m4

```diff
@@ -21,6 +21,62 @@ include(`pga.m4')
 include(`muxdemux.m4')
 include(`mixercontrol.m4')
 include(`bytecontrol.m4')
+include(`dai.m4')
```

Comment:
for this file i think it might be better to just copy this file and make a "waves-demux" pipeline